<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QWidget" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gwint - Ustawienia</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/witcher.png</normaloff>:/icons/witcher.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ustawienia</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditNick">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ako</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Cards</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nick</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="comboBoxKarty">
        <item>
         <property name="text">
          <string>Elfy</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/ikony/ikona3.png</normaloff>:/ikony/ikona3.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Krolestwo Polnocy</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/ikony/ikona1.png</normaloff>:/ikony/ikona1.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nilfgaard</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/ikony/ikona2.png</normaloff>:/ikony/ikona2.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Potwory</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/ikony/ikona4.png</normaloff>:/ikony/ikona4.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxRete">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Network</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="4">
       <widget class="QRadioButton" name="radioButtonServer">
        <property name="text">
         <string>Server</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelServerPort">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBoxServerPort">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>6667</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QRadioButton" name="radioButtonClient">
        <property name="text">
         <string>Client</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelClientIp">
        <property name="text">
         <string>IP address</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>3</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEditClientIp">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelClientPort">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="spinBoxClientPort">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>6667</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>248</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEsci">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/cards.qrc">
         <normaloff>:/ikony/nie.png</normaloff>:/ikony/nie.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/cards.qrc">
         <normaloff>:/ikony/tak.png</normaloff>:/ikony/tak.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/cards.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEditNick</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonOk</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
